src/containerapp/azext_containerapp/containerapp_resiliency_decorator.py

```diff
@@ -0,0 +1,476 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from typing import Dict, Any
+
+from azure.cli.core.azclierror import (ValidationError, ResourceNotFoundError)
+from azure.cli.core.commands import AzCliCommand
+from msrest.exceptions import DeserializationError
+
+from ._decorator_utils import load_yaml_file, create_deserializer, process_containerapp_resiliency_yaml
```

Comment:
should we use the create_deserializer function in core cli? Is there any difference? @Greedygre

Reply:
In fact they have no difference.